src/celeritas/optical/CerenkovPreGenerator.hh

```diff
@@ -26,12 +27,13 @@ namespace celeritas
  * \c CerenkovGenerator and populate \c OpticalDistributionData values using
  * Param and State data.
  * \code
-    CerenkovPreGenerator::OpticalPreGenStepData step_data;
```

Comment:
I guess we could clean up and remove this comment code block? (Which is my fault, since I sketched an idea on how to use it when we didn't discuss much yet.) Also the `ScintillationPreGenerator` doxygen does not have an equivalent one, and I guess it's going to be self-explanatary when we actually have it in the main loop... thoughts?